Makefile

```diff
@@ -2,10 +2,18 @@ PROG=	bsdec2-image-upload
 SRCS=	main.c
 MAN	=
 WARNS	?=	3
-CFLAGS	+=	-Wno-error=\#warnings
 BINDIR	?=	/usr/local/bin
 LDADD	+=	-lcrypto -lssl
 
+UNAME_S	:=	$(shell uname -s)
+.if (${UNAME_S},Linux)
+CERTFILE	=	\"/etc/ssl/certs/ca-bundle.crt\"
+CFLAGS	+=	--no-warnings
```

Comment:
Still not sure why you're disabling all compiler warnings on Linux.

Reply:
What does -Wno-error=\#warnings do? That was my best guess as to what it did.